manim/mobject/svg/text_mobject.py

```diff
@@ -11,14 +11,15 @@
 
     class HelloWorld(Scene):
         def construct(self):
-            text = Text('Hello world').scale(3)
+            text = Text('Hello world')
```

Comment:
are you saying that you automatically scale this up to fit the screen?
What happens if I keep the `scale(3)`?

Reply:
![scale-3](https://user-images.githubusercontent.com/49693820/115605859-aa742d00-a300-11eb-91ba-47663af7ffda.png)
^ when scale=3

when there is no scale it generates

![HelloWorld_ManimCE_v0 5 0](https://user-images.githubusercontent.com/49693820/115605965-c677ce80-a300-11eb-8475-4b9be4e3f00c.png)

Reply:
ah. well may I ask: why did you remove the scale(3) from the example?